samples/Infrastructure/Storage.fs

```diff
@@ -76,7 +76,7 @@ module Cosmos =
         let conn = connector.Connect("equinox-tool", discovery) |> Async.RunSynchronously
         let cacheStrategy =
             if cache then
-                let c = Caching.Cache("equinox-tool", sizeMb = 50)
+                let c = Equinox.Cache("equinox-tool", sizeMb = 50)
```

Comment:
now this is more general, the sample can centralize the creation of the cache. Doesn't have to happen in this PR though

Reply:
slightly scared to touch it in scope of this PR. Will leave it as is for now if you don't mind.

Reply:
that's reasonable - small PRs are good PRs too